app/Repositories/LocationRepository.php

```diff
@@ -105,8 +108,14 @@ public function analystic($full_time_seat, $shiftData, $filter, $title, $classNa
             } else {
                 $count = 0;
             }
-            if (array_key_exists($date, $full_time_seat)) {
-                $count += $full_time_seat[$date];
+            if (array_key_exists($date, $fullTimeSeat)) {
+                $count += $fullTimeSeat[$date];
+            }
+            $count = $totalSeat - $count;
+            if ($count < 0) {
+                $count = __('lacking') . $count * -1;
+            } else if ($count >= 0) {
+                $count = __('over') . $count;
             }
             $data[] = [
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.ControlStructures.ElseIfDeclaration.NotAllowed</br>**Severity**: warning</br>**File**: app/Repositories/LocationRepository.php</br>Usage of ELSE IF is discouraged; use ELSEIF instead